### ISSUE TITLE
Unable to load file

### ISSUE THREAD

**Issue by gergog76** [joaomoreno/gulp-atom-electron#37]
Hi,

I cannot package my electron app. I always got this error message: 

```
stream.js:74
      throw er; // Unhandled stream error in pipe.
      ^
Fatal error: Unable to load file

```

The relevant gulp task:

```
gulp.task('package:windows', function() {
  return gulp.src(config.buildDir + '/**')
  .pipe(debug({title: 'before:'}))
  .pipe(electron({
      version: '1.4.3',
      platform: 'win32',
      arch: 'x64'
    }))
//    .pipe(debug({title: 'after:'}))
    .pipe(symdest(config.packagesDir + '/windows'));
});

```

I run out of ideas howto debug it ny further.

Any idea?

Thanks!

**Comment by joaomoreno** [comment]
Strange. Can you maybe delete the `gulp-atom-electron` folder in your TMPDIR?

**Comment by ChristopherBock** [comment]
I ran into the very same problem. It is a file-system racing condition. This should be fixed by the linked PR. Instead of running rcedit when the stream's finish event is triggered, it should be run when the close event is triggered, see e.g. http://stackoverflow.com/questions/13156243/event-associated-with-fs-createwritestream-in-node-js .

**Comment by joaomoreno** [comment]
Fixed by #38 in 1.10.0
